app/controllers/bigbluebutton_api_controller.rb

```diff
@@ -379,8 +377,14 @@ def recordings_disabled
 
   # Filter out unneeded params when passing through to join and create calls
   # Has to be to_unsafe_hash since to_h only accepts permitted attributes
-  def pass_through_params
-    params.except(:format, :controller, :action, :checksum).to_unsafe_hash
+
+  def pass_through_params(name)
+    if ENV["#{name}_EXCLUDE_PARAMS"].nil?
+      params.except(:format, :controller, :action, :checksum).to_unsafe_hash
+    else
+      params.except(*(ENV["#{name}_EXCLUDE_PARAMS"].split(',').map(&:strip) +
```

Comment:
Rather than parse the environment variables at runtime on every request, please parse them to two separate config variables - e.g. `config.x.create_exclude_params` and `config.x.join_exclude_params` in `config/application.rb`.